src/app/services/Services.ts

```diff
@@ -124,7 +124,7 @@ export abstract class ProblemService {
 })
 export abstract class ExamService {
 
-  abstract getExamProgressOverview(studentId: number, examId: number): Observable<ExamOverview>;
+  abstract getExamProgressOverview(studentId: number, examId: number, cached?: boolean): Observable<ExamOverview>;
```

Comment:
I think we don't need to cache it here, so just leave the non-cache implementation.